modules/options-for-notebook-server-environments.adoc

```diff
@@ -285,6 +285,22 @@ a| * Python 3.8
 * PyTorch 2.0
 * Elyra 3.15
 
+| Code Server
+| 2023.2 (Recommended)
+a| * Python 3.9
+* Boto3 1.29
+* Kafka-Python 2.0
+* Matplotlib 3.8
+* Numpy 1.26
+* Pandas 2.1
+* Plotly 5.18
+* Scikit-learn 1.3
+* Scipy 1.11
+* Skl2onnx 1.15
```

Comment:
Please use the name that we have also for other images to keep this consistent :upside_down_face: 
```suggestion
* Sklearn-onnx 1.15
```

Reply:
Updated, thank you!